frontend/src/components/JoinRoom.tsx

```diff
@@ -0,0 +1,88 @@
+import { Button, Container, Flex, FormLabel, Heading, Input, InputGroup, Stack, useToast } from "@chakra-ui/react";
+import BaseHelper                                                                          from "../helpers/BaseHelper";
+import { useState }                                                                        from "react";
+import axios                                                                               from "axios";
+import { JoinRoomState }                                                                   from "../interfaces/RoomInterface";
+
+const JoinRoom = () => {
+    const toast = useToast();
+
+    const [getState, setState] = useState<JoinRoomState>({
+        roomUrl: '',
+        password: ''
+    });
+
+    const joinRoom = () => {
+        const url = BaseHelper.generateUrl('join-room');
+        const formData = BaseHelper.buildFormData(getState);
+
+        axios.post(url, formData)
+            .then(res => {
+                console.log('RESPONSE', res.data);
+                // @ts-ignore
```

Comment:
Same as above comment, would setting `any` type to `res` remove the need for the `ts-ignore`?